man/fi_av.3.md

```diff
@@ -70,6 +70,9 @@ int fi_av_insert_auth_key(struct fid_av *av, const void *auth_key,
 
 int fi_av_lookup_auth_key(struct fid_av *av, fi_addr_t addr,
       void *auth_key, size_t *auth_key_size);
+
+int fi_av_set_user_id(struct fid_av *av, fi_addr_t fi_addr,
+      fi_addr_t user_id);
```

Comment:
I _think_ a flags option with FI_AUTH_KEY support needs to be added. This would allow a user-defined authorization key src_addr to be return in CQ error event.